app/actions/websocket/index.ts

```diff
@@ -398,3 +409,40 @@ export async function handleEvent(serverUrl: string, msg: WebSocketMessage) {
             break;
     }
 }
+
+async function fetchPostDataIfNeeded(serverUrl: string) {
+    try {
+        const {database} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+        const currentChannelId = await getCurrentChannelId(database);
+        const isCRTEnabled = await getIsCRTEnabled(database);
+        const mountedScreens = NavigationStore.getScreensInStack();
+        const isChannelScreenMounted = mountedScreens.includes(Screens.CHANNEL);
+        const tabletDevice = await isTablet();
+
+        if (isCRTEnabled && NavigationStore.getVisibleScreen() === Screens.THREAD) {
```

Comment:
Should be the visible screen, or mounted screen? Mainly if we have a screen on top of this one, we still want to update, right?

Reply:
Also, shouldn't we mark the thread as read somewhere?

Reply:
Marking the thread as read is handled in the component.

Thanks, Making the change to the mounted screen.